layers/+tools/lsp/packages.el

```diff
@@ -11,10 +11,12 @@
 
 (defconst lsp-packages
   '(
-     lsp-mode
-     lsp-ui
-     (company-lsp :requires company)
-     ))
+    lsp-mode
+    lsp-ui
+    (company-lsp :requires company)
+    helm-lsp
+    lsp-treemacs
```

Comment:
I think this one should be `(lsp-treemacs :requires treemacs)`.

Reply:
Is this really needed? I tested with setting `neotree` as a file explorer and it seems to be working fine. lsp-treemacs uses treemacs only as a tree renderer which doesn't seem to interfere with file browser functionality. I think that is different from the company case since company-lsp is directly related to completion framework. What do you think? It will be a pity if we have this functionality only if the user has selected treemacs as a file browser.

Reply:
OK, if it's not conflicting with other file explorers then all is good! :+1: